torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -1038,8 +1150,59 @@ def generate_detector_report(self, model: GraphModule) -> Tuple[str, Dict[str, A
             String report of of whether there are outliers in the activations around certain modules
             Dictionary mapping modules of interest to:
                 whether there were outliers found in activation before
+                the number of batches used for each channel
+                whether the number of applicable batches is above statistical threshold (30)
```

Comment:
would maybe change language because its a kind of arbitrary threshold, we probably don't want to imply statistical significance without knowing anything about relevant distributions.

Reply:
updated to just say above a minimum set threshold so not misleading